templates/credentials/your-exams.html

```diff
@@ -27,6 +29,11 @@ <h1>Your exams</h1>
         </tr>
       </thead>
       <tbody>
+        {% if exam|length == 0 %}
```

Comment:
I think this should be `exams` instead, because I see the message even after activating keys.